swagger.yaml

```diff
@@ -3766,6 +3792,19 @@ definitions:
       - 'master'
       - 'worker'
       - 'bootstrap'
+  
+  disk-role:
+    type: string
+    enum:
+      - 'selected'
+      - 'bootable'
```

Comment:
Disks [already](https://github.com/openshift/assisted-service/blob/bab0090a96a2397efcec070ed69b95a1bd2583b4/swagger.yaml#L3624-L3625) have a "bootable" attribute:

Reply:
This is for the content of role column, since there can be several bootable disks but only one selected. It awaits the confirmation of UX as well.

Reply:
I'm not sure what bootable and non-bootable mean.  If I have a bootable disk and mark it as non-bootable it will cause the disk to be formatted?  And if I leave it as bootable we won't format?  What if I have a non-bootable disk and I mark it as bootable?  What happens if I choose 2 disks to be "selected"?  This API is confusing.  I expected each host to have properties called "installation_disk" and "installation_network".  Am I missing something?